pkg/protocompat/time_test.go

```diff
@@ -208,6 +229,13 @@ func TestDurationFromProto(t *testing.T) {
 	timeDuration, err := DurationFromProto(protoDuration)
 	assert.NoError(t, err)
 	assert.Equal(t, expectedDuration, timeDuration)
+
+	protoInvalidDuration := &types.Duration{
+		Seconds: 253402300801,
```

Comment:
Where does this value come from ? It looks like it is the number of seconds from the Unix Epoch to the first invalid timestamp second.

Reply:
Do we want to test the various Duration validation cases ?
- Seconds below minSeconds (corrected for overflow)
- Seconds above maxSeconds (corrected for overflow)
- Nanos below -999999999
- Nanos above 999999999
- Sign difference between seconds and nanos (in both directions)
- Resulting duration is not of the same sign as input duration Nanos

Reply:
I have added additional information in the description. The intention is not to cover all validation edge cases but to ensure that an error will be returned for an invalid value.